docs/source/user-guide/integrations/h2o.rst

```diff
@@ -0,0 +1,171 @@
+Using deepchecks with an H2O Model
```

Comment:
```suggestion
============================
H2O
```
This title will show up on the Integration table of content, so it should be short and clear.